docs/coding-guidelines/new-functions.md

```diff
@@ -0,0 +1,30 @@
+Minium requirements for implementing Spark API functions
```

Comment:
Many thanks for writing this down! :)

Reply:
no problem, I just thought it might help other people :)